src/universal-dapp.js

```diff
@@ -481,31 +483,8 @@ UniversalDApp.prototype.getCallButton = function (args) {
             } else if (args.abi.constant && !isConstructor) {
                 replaceOutput($result, getReturnOutput(result));
             } else {
-
-                function tryTillResponse (txhash, done) {
-                    this.web3.eth.getTransactionReceipt(result, testResult);
-
-                    function testResult (err, address) {
-                        if (!err && !address) {
-                            setTimeout(function () { tryTillResponse(txhash, done); }, 500);
-                        } else {
-                            done(err, address);
-                        }
-                    }
-
-                }
-                tryTillResponse(result, function (err, result) {
```

Comment:
Why did you remove this call to `tryTillResponse`? It seems like that would change the behavior (in a bad way).